pkgs/os-specific/linux/ebtables/default.nix

```diff
@@ -12,16 +23,28 @@ stdenv.mkDerivation rec {
   makeFlags =
     [ "LIBDIR=$(out)/lib" "BINDIR=$(out)/sbin" "MANDIR=$(out)/share/man"
       "ETCDIR=$(out)/etc" "INITDIR=$(TMPDIR)" "SYSCONFIGDIR=$(out)/etc/sysconfig"
-      "LOCALSTATEDIR=/var"
+      "LOCALSTATEDIR=/var" "CC=${stdenv.cc}/bin/${stdenv.cc.targetPrefix}cc"
```

Comment:
We' have issues with this in the past. Sometimes packages will save $CC in their output for debugging, causing a dependency on the compiler. To avoid that, you should just do ```CC=${stdenv.cc.targetPrefix}cc```.